### PR TITLE
wmfocus: init at 1.0.2

### PR DESCRIPTION
**Pull request by Synthetica9**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/applications/window-managers/i3/wmfocus.nix [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  pname = "wmfocus";
+  version = "1.0.2";
+in
+rustPlatform.buildRustPackage {
+  inherit pname version;
+  name = "${pname}-${version}";
+
+  buildInputs = [python3 pkgconfig glib xorg.libX11 cmake cairo xorg.libX11];
+  nativeBuildInputs = [libxkbcommon xorg.libxcb xorg.xcbutilkeysyms];
+
```
Comment (xeji) [inline pkgs/applications/window-managers/i3/wmfocus.nix]: These `buildInputs` and `nativeBuildInputs` don't look right. `nativeBuildInputs` should only contain packages that need to run at buildtime (here: `pkgconfig` and `python3`). The rest should be `buildInputs`. Some of them, like `cmake` aren't needed at all. I found this to be sufficient to build the package (and it seems to run fine):

```
nativeBuildInputs = [ python3 pkgconfig ];
buildInputs = [ cairo libxkbcommon xorg.xcbutilkeysyms ];
```

These `buildInputs` also match the upstream list of dependencies (minus rust and cargo, which are implied by buildRustPackage).

**Review by xeji — COMMENTED** [review]
Nice feature! The buildInputs needs fixing, LGTM otherwise.

**Comment by Synthetica9** [comment]
cc @xeji 2255419

File: pkgs/applications/window-managers/i3/wmfocus.nix (hunk 1 above)
Comment (Synthetica9) [reply pkgs/applications/window-managers/i3/wmfocus.nix]: @xeji 2255419

**Pull request merged**